versions.txt

```diff
@@ -2,10 +2,10 @@
 # by default with the OpenTelemetry Operator. This would usually be the latest
 # stable OpenTelemetry version. When you update this file, make sure to update the
 # the docs as well.
-opentelemetry-collector=0.60.0
+opentelemetry-collector=0.61.0
 
 # Represents the current release of the OpenTelemetry Operator.
-operator=0.60.0
+operator=0.61.0
 
 # Represents the current release of the Target Allocator.
 targetallocator=0.1.0
```

Comment:
@yuriolisa can you also bump the target allocator version please?

```suggestion
targetallocator=0.61.0
```

Reply:
We do not bump the target allocator version in line with operator releases. It's always the `0.1.0` as we treat the target allocator as a subsystem with its own versioning. 

If there are any major improvements/breaking changes we can bump it with minor/patches release versions.

Reply:
That has changed https://github.com/open-telemetry/opentelemetry-operator/blob/main/RELEASE.md.

@jaronoff97 maybe there should be a single version for operator and TA. The TA image could be released if that version changes.